### ISSUE TITLE
Installing LiipImagineBundle for Symfony 2.7 using Composer 

### ISSUE THREAD

**Issue by Hellow** [liip/LiipImagineBundle#902]
Hi !

I'm trying to install the LiipImagineBundle using Composer on my Symfony project 2.7 but I  keep getting the same errors

>  composer require liip/imagine-bundle

```
Using version ^1.7 for liip/imagine-bundle
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package sensio/geneframeworkrator-bundle could not be found in any version, there may be a typo in the package name.
  Problem 2
    - Conclusion: remove symfony/symfony v2.7.24
    - Conclusion: don't install symfony/symfony v2.7.24
    - symfony/framework-bundle v2.8.0 requires symfony/translation ~2.8 -> satisfiable by symfony/translation[v2.8.0, v2.8.1, v2.8.10, v2.8.11, v2.8.12, v2.8.13, v2.8.14, v2.8.15, v2.8.16, v2.8.17, v2.8.18, v2.8.2, v2.8.3, v2.8.4, v2.8.5, v2.8.6, v2.8.7, v2.8.8, v2.8.9].
    - symfony/framework-bundle v2.8.1 requires symfony/translation ~2.8 -> satisfiable by symfony/translation[v2.8.0, v2.8.1, v2.8.10, v2.8.11, v2.8.12, v2.8.13, v2.8.14, v2.8.15, v2.8.16, v2.8.17, v2.8.18, v2.8.2, v2.8.3, v2.8.4, v2.8.5, v2.8.6, v2.8.7, v2.8.8, v2.8.9].
    - symfony/framework-bundle v2.8.10 requires symfony/translation ~2.8 -> satisfiable by symfony/translation[v2.8.0, v2.8.1, v2.8.10, v2.8.11, v2.8.12, v2.8.13, v2.8.14, v2.8.15, v2.8.16, v2.8.17, v2.8.18, v2.8.2, v2.8.3, v2.8.4, v2.8.5, v2.8.6, v2.8.7, v2.8.8, v2.8.9].
    - symfony/framework-bundle v2.8.11 requires symfony/translation ~2.8 -> satisfiable by symfony/translation[v2.8.0, v2.8.1, v2.8.10, v2.8.11, v2.8.12, v2.8.13, v2.8.14, v2.8.15, v2.8.16, v2.8.17, v2.8.18, v2.8.2, v2.8.3, v2.8.4, v2.8.5, v2.8.6, v2.8.7, v2.8.8, v2.8.9].
    - symfony/framework-bundle v2.8.12 requires symfony/translation ~2.8 -> satisfiable by symfony/translation[v2.8.0, v2.8.1, v2.8.10, v2.8.11, v2.8.12, v2.8.13, v2.8.14, v2.8.15, v2.8.16, v2.8.17, v2.8.18, v2.8.2, v2.8.3, v2.8.4, v2.8.5, v2.8.6, v2.8.7, v2.8.8, v2.8.9].
    - symfony/framework-bundle v2.8.13 requires symfony/translation ~2.8 -> satisfiable by symfony/translation[v2.8.0, v2.8.1, v2.8.10, v2.8.11, v2.8.12, v2.8.13, v2.8.14, v2.8.15, v2.8.16, v2.8.17, v2.8.18, v2.8.2, v2.8.3, v2.8.4, v2.8.5, v2.8.6, v2.8.7, v2.8.8, v2.8.9].
    - symfony/framework-bundle v2.8.14 requires symfony/translation ~2.8 -> satisfiable by symfony/translation[v2.8.0, v2.8.1, v2.8.10, v2.8.11, v2.8.12, v2.8.13, v2.8.14, v2.8.15, v2.8.16, v2.8.17, v2.8.18, v2.8.2, v2.8.3, v2.8.4, v2.8.5, v2.8.6, v2.8.7, v2.8.8, v2.8.9].
    - symfony/framework-bundle v2.8.15 requires symfony/translation ~2.8 -> satisfiable by symfony/translation[v2.8.0, v2.8.1, v2.8.10, v2.8.11, v2.8.12, v2.8.13, v2.8.14, v2.8.15, v2.8.16, v2.8.17, v2.8.18, v2.8.2, v2.8.3, v2.8.4, v2.8.5, v2.8.6, v2.8.7, v2.8.8, v2.8.9].
    - symfony/framework-bundle v2.8.16 requires symfony/translation ~2.8 -> satisfiable by symfony/translation[v2.8.0, v2.8.1, v2.8.10, v2.8.11, v2.8.12, v2.8.13, v2.8.14, v2.8.15, v2.8.16, v2.8.17, v2.8.18, v2.8.2, v2.8.3, v2.8.4, v2.8.5, v2.8.6, v2.8.7, v2.8.8, v2.8.9].
    - symfony/framework-bundle v2.8.17 requires symfony/translation ~2.8 -> satisfiable by symfony/translation[v2.8.0, v2.8.1, v2.8.10, v2.8.11, v2.8.12, v2.8.13, v2.8.14, v2.8.15, v2.8.16, v2.8.17, v2.8.18, v2.8.2, v2.8.3, v2.8.4, v2.8.5, v2.8.6, v2.8.7, v2.8.8, v2.8.9].
    - symfony/framework-bundle v2.8.18 requires symfony/translation ~2.8 -> satisfiable by symfony/translation[v2.8.0, v2.8.1, v2.8.10, v2.8.11, v2.8.12, v2.8.13, v2.8.14, v2.8.15, v2.8.16, v2.8.17, v2.8.18, v2.8.2, v2.8.3, v2.8.4, v2.8.5, v2.8.6, v2.8.7, v2.8.8, v2.8.9].
    - symfony/framework-bundle v2.8.2 requires symfony/translation ~2.8 -> satisfiable by symfony/translation[v2.8.0, v2.8.1, v2.8.10, v2.8.11, v2.8.12, v2.8.13, v2.8.14, v2.8.15, v2.8.16, v2.8.17, v2.8.18, v2.8.2, v2.8.3, v2.8.4, v2.8.5, v2.8.6, v2.8.7, v2.8.8, v2.8.9].
    - symfony/framework-bundle v2.8.3 requires symfony/translation ~2.8 -> satisfiable by symfony/translation[v2.8.0, v2.8.1, v2.8.10, v2.8.11, v2.8.12, v2.8.13, v2.8.14, v2.8.15, v2.8.16, v2.8.17, v2.8.18, v2.8.2, v2.8.3, v2.8.4, v2.8.5, v2.8.6, v2.8.7, v2.8.8, v2.8.9].
    - symfony/framework-bundle v2.8.4 requires symfony/translation ~2.8 -> satisfiable by symfony/translation[v2.8.0, v2.8.1, v2.8.10, v2.8.11, v2.8.12, v2.8.13, v2.8.14, v2.8.15, v2.8.16, v2.8.17, v2.8.18, v2.8.2, v2.8.3, v2.8.4, v2.8.5, v2.8.6, v2.8.7, v2.8.8, v2.8.9].
    - symfony/framework-bundle v2.8.5 requires symfony/translation ~2.8 -> satisfiable by symfony/translation[v2.8.0, v2.8.1, v2.8.10, v2.8.11, v2.8.12, v2.8.13, v2.8.14, v2.8.15, v2.8.16, v2.8.17, v2.8.18, v2.8.2, v2.8.3, v2.8.4, v2.8.5, v2.8.6, v2.8.7, v2.8.8, v2.8.9].
    - symfony/framework-bundle v2.8.6 requires symfony/translation ~2.8 -> satisfiable by symfony/translation[v2.8.0, v2.8.1, v2.8.10, v2.8.11, v2.8.12, v2.8.13, v2.8.14, v2.8.15, v2.8.16, v2.8.17, v2.8.18, v2.8.2, v2.8.3, v2.8.4, v2.8.5, v2.8.6, v2.8.7, v2.8.8, v2.8.9].
    - symfony/framework-bundle v2.8.7 requires symfony/translation ~2.8 -> satisfiable by symfony/translation[v2.8.0, v2.8.1, v2.8.10, v2.8.11, v2.8.12, v2.8.13, v2.8.14, v2.8.15, v2.8.16, v2.8.17, v2.8.18, v2.8.2, v2.8.3, v2.8.4, v2.8.5, v2.8.6, v2.8.7, v2.8.8, v2.8.9].
    - symfony/framework-bundle v2.8.8 requires symfony/translation ~2.8 -> satisfiable by symfony/translation[v2.8.0, v2.8.1, v2.8.10, v2.8.11, v2.8.12, v2.8.13, v2.8.14, v2.8.15, v2.8.16, v2.8.17, v2.8.18, v2.8.2, v2.8.3, v2.8.4, v2.8.5, v2.8.6, v2.8.7, v2.8.8, v2.8.9].
    - symfony/framework-bundle v2.8.9 requires symfony/translation ~2.8 -> satisfiable by symfony/translation[v2.8.0, v2.8.1, v2.8.10, v2.8.11, v2.8.12, v2.8.13, v2.8.14, v2.8.15, v2.8.16, v2.8.17, v2.8.18, v2.8.2, v2.8.3, v2.8.4, v2.8.5, v2.8.6, v2.8.7, v2.8.8, v2.8.9].
    - don't install symfony/translation v2.8.0|don't install symfony/symfony v2.7.24
    - don't install symfony/translation v2.8.1|don't install symfony/symfony v2.7.24
    - don't install symfony/translation v2.8.10|don't install symfony/symfony v2.7.24
    - don't install symfony/translation v2.8.11|don't install symfony/symfony v2.7.24
    - don't install symfony/translation v2.8.12|don't install symfony/symfony v2.7.24
    - don't install symfony/translation v2.8.13|don't install symfony/symfony v2.7.24
    - don't install symfony/translation v2.8.14|don't install symfony/symfony v2.7.24
    - don't install symfony/translation v2.8.15|don't install symfony/symfony v2.7.24
    - don't install symfony/translation v2.8.16|don't install symfony/symfony v2.7.24
    - don't install symfony/translation v2.8.17|don't install symfony/symfony v2.7.24
    - don't install symfony/translation v2.8.18|don't install symfony/symfony v2.7.24
    - don't install symfony/translation v2.8.2|don't install symfony/symfony v2.7.24
    - don't install symfony/translation v2.8.3|don't install symfony/symfony v2.7.24
    - don't install symfony/translation v2.8.4|don't install symfony/symfony v2.7.24
    - don't install symfony/translation v2.8.5|don't install symfony/symfony v2.7.24
    - don't install symfony/translation v2.8.6|don't install symfony/symfony v2.7.24
    - don't install symfony/translation v2.8.7|don't install symfony/symfony v2.7.24
    - don't install symfony/translation v2.8.8|don't install symfony/symfony v2.7.24
    - don't install symfony/translation v2.8.9|don't install symfony/symfony v2.7.24
    - Installation request for symfony/symfony (locked at v2.7.24, required as 2.7.*) -> satisfiable by symfony/symfony[v2.7.24].
    - Installation request for symfony/framework-bundle ^2.8 -> satisfiable by symfony/framework-bundle[v2.8.0, v2.8.1, v2.8.10, v2.8.11, v2.8.12, v2.8.13, v2.8.14, v2.8.15, v2.8.16, v2.8.17, v2.8.18, v2.8.2, v2.8.3, v2.8.4, v2.8.5, v2.8.6, v2.8.7, v2.8.8, v2.8.9].

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://getcomposer.org/doc/04-schema.md#minimum-stability> for more details.

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.
```

I also tried to install older version of the bundle but still getting the same error, for example:

> composer require liip/imagine-bundle "1.5"

```
Problem 1
    - The requested package sensio/geneframeworkrator-bundle could not be found in any version, there may be a typo in the package name.
  Problem 2
    - Installation request for liip/imagine-bundle 1.5 -> satisfiable by liip/imagine-bundle[1.5.0].
    - Conclusion: remove symfony/symfony v2.7.24
    - Conclusion: don't install symfony/symfony v2.7.24
    - liip/imagine-bundle 1.5.0 requires symfony/framework-bundle ~2.3|~3.0 -> satisfiable by symfony/framework-bundle[v2.8.0, v2.8.1, v2.8.10, v2.8.11, v2.8.12, v2.8.13, v2.8.14, v2.8.15, v2.8.16, v2.8.17, v2.8.18, v2.8.2, v2.8.3, v2.8.4, v2.8.5, v2.8.6, v2.8.7, v2.8.8, v2.8.9].

.... (same error as before)......
```

Do I do it wrong ?
Please enlighten me, thanks !

**Comment by robfrawley** [comment]
@Hellow The error you are getting is unrelated to this bundle, but let's see if we can't fix your Composer dependencies issue, regardless.

The error Composer is displaying is pretty clear: the bundle `sensio/geneframeworkrator-bundle` cannot be found in any version, ie **that bundle does not exist**. You seem to be trying to require a combination of the following bundles:

- `sensio/generator-bundle`
- `sensio/framework-extra-bundle`

To be clear, _the bundle mentioned in your Composer error message `sensio/geneframeworkrator-bundle`, does not exist._ If these changes don't help resolve your issue, include your complete `composer.json` file in subsequent replies.

**Comment by Hellow** [comment]
Yes, I understood it a little bit later after removing the vendor repo and re-installing the dependencies with composer.

It's in fact a problem with my version of Symfony, don't know why suddenly the package "symfony/symfony 2.7.*" is blocked like that. I tried everything but... I found no solution, it's definitely blocked.
I had to change it from 2.7.* to 2.8. I hope I will not have more problems that I already had after updated it.

Sorry for the unrelated problem and thank you !
